boards/infineon/cyw920829m2evk_02/cyw920829m2evk_02.dts

```diff
@@ -24,6 +24,26 @@
 		zephyr,shell-uart = &uart2;
 		zephyr,bt-hci = &bluetooth;
 	};
+
+	power-states {
+		idle: idle {
+			compatible = "zephyr,power-state";
+			power-state-name = "suspend-to-idle";
+			min-residency-us = <1000000>;
+		};
+
+		suspend_to_ram: suspend_to_ram {
+			compatible = "zephyr,power-state";
+			power-state-name = "suspend-to-ram";
+			min-residency-us = <2000000>;
+		};
+	};
+
+	cpus {
+		cpu@0 {
+			cpu-power-states = <&idle &suspend_to_ram>;
+		};
+	};
```

Comment:
power states need to be part of soc dts files

Reply:
moved the power states to soc dtsi files..

Reply:
@gmarull, please review Sreeram's change at your convenience.